javascripts/2048.js

```diff
@@ -41,18 +47,15 @@ function newTile(openCells, usedCells) {
   $(tile).attr("data-val", initNum);
   $(tile).text(initNum);
   occupyCell(openCells, usedCells, cellSpace);
-  // emptyCell(openCells, usedCells, cellSpace)
-
 }
 
 function occupyCell(openCells, usedCells, cellSpace) {
   var index = findIndex(openCells, cellSpace);
   openCells.splice(index, 1);
-  var cellExists = findIndex(usedCells, cellSpace)
+  var cellExists = findIndex(usedCells, cellSpace);
   if (cellExists == undefined) {
```

Comment:
At first I didn't have this line, but then I noticed that some how there were duplicates in usedCells, so I added this check.